src/contact-us.njk

```diff
@@ -2,6 +2,8 @@
 layout: layouts/nohero.njk
 title: Contact us
 sitemapPriority: 0.8
+meta:
+    description: Get in touch with 
```

Comment:
Is that intentionally cut off mid-sentence so if gets joined with 'FlowForge' in the title?

Reply:
nope, good spot, that's an error